docs/conf.py

```diff
@@ -28,9 +28,8 @@
 # Add any Sphinx extension module names here, as strings. They can be
```

Comment:
Unrelated to the line, but opening a thread to isolate messaging.

While removing `m2r`, we observed multiple `make html` warnings and errors.  I investigated to discover the branch behavior is _no worse_ than `master`.   While undesirable, since the behavior is no worse, I propose addressing the warnings and errors out-of-band to the PR.

Here's what I did:
* Capture `make html` output from `master` and the branch:  `make html >>/tmp/make_html_pyteal_docs_readme.txt 2>&1`.
* Diff warnings and errors.   The output shows the branch produces 2 _fewer_ warnings and the _same_ errors.
```
(.venv) michael@albode docs % diff <(grep -iE 'warning|error' /tmp/make_html_master.txt) <(grep -iE 'warning|error' /tmp/make_html_pyteal_docs_readme.txt)
1d0
< WARNING: html_static_path entry '_static' does not exist
14d12
< /Users/michael/dev/pyteal/docs/.venv/lib/python3.10/site-packages/idna-3.3.dist-info/LICENSE.md: WARNING: document isn't included in any toctree
22c20
< build succeeded, 21 warnings.
---
> build succeeded, 19 warnings.
```
* Here's the source files in case you wish to review.
  * [make_html_master.txt](https://github.com/algorand/pyteal/files/8128930/make_html_master.txt)
  * [make_html_pyteal_docs_readme.txt](https://github.com/algorand/pyteal/files/8128929/make_html_pyteal_docs_readme.txt)